scripts/live_test/template.yml

```diff
@@ -59,20 +62,20 @@ jobs:
     - task: PublishTestResults@2
       condition: succeededOrFailed()
       inputs:
-        testResultsFiles: '/home/vsts/.azdev/env_config/home/vsts/work/1/env/test_results.xml'
+        testResultsFiles: '/mnt/vss/.azdev/env_config/mnt/vss/_work/1/env/test_results.xml'
```

Comment:
`azdev` saves its config to `~/.azdev`:

https://github.com/Azure/azure-cli-dev-tools/blob/2de5818f37a741a1ae7765ca43126ac2b00d622f/azdev/utilities/config.py#L25

so `/mnt/vss` should be available as `~` or `$HOME`.